dhis-2/dhis-web/dhis-web-dataentry/src/main/java/org/hisp/dhis/de/action/LoadFormAction.java

```diff
@@ -505,4 +506,13 @@ private void addOptionAccess( User user, Map<String, Boolean> optionAccessMap,
             }
         } );
     }
+
+    private List<CategoryCombo> getSortedCategoryCombos( Set<CategoryCombo> categoryCombos )
+    {
+        List<CategoryCombo> listCategoryCombos = new ArrayList<>( categoryCombos );
```

Comment:
Do we need to make another list here? Could we just use the original `categoryCombos` list?